src/torch_models.py

```diff
@@ -278,7 +279,27 @@ def _normalize_data(data: Tensor) -> Tuple[Tensor, Tensor, Tensor]:
         return (data - shift) / scale, shift, scale
 
 
-class MLPClassifier(nn.Module):
+class Classifier(abc.ABC):
+    """ABC for classifier types."""
+
+    @abc.abstractmethod
+    def fit(self, X: Array, y: Array) -> None:
+        """Train classifier on the given data.
```

Comment:
Sorry to revive a merged PR! @amburger66 could you remove the duplicate docstrings on the fit() and classify() in MLPClassifier below? Linter won't complain bc you don't need a docstring for overriden methods, and so it's better to have fewer duplicate lines when possible.

Reply:
my bad, will do!